Dockerfile

```diff
@@ -10,11 +9,12 @@ RUN apk add git musl-dev curl
 ARG SHELL=/bin/ash
 RUN curl --proto '=https' --tlsv1.2 -sSf https://docs.swmansion.com/scarb/install.sh | $SHELL -s -- -v $SCARB_VERSION
 
-# Install cairo1 compiler
-RUN git clone -b $CAIRO_COMPILER_TARGET_TAG https://github.com/starkware-libs/cairo.git
+ARG COMPILER_BINARY_URL
+ARG ASSET_NAME
 
-RUN cargo build --release --manifest-path /cairo/crates/cairo-lang-starknet/Cargo.toml
-RUN cargo build --release --manifest-path /cairo/crates/cairo-lang-sierra-to-casm/Cargo.toml
+# Download cairo1 compiler
+RUN curl --location -O --request GET $COMPILER_BINARY_URL
+RUN tar -zxvf $ASSET_NAME
 
 # Install cairo-lang
 FROM python:3.9.13-alpine3.16 as cairo-lang
```

Comment:
Can we remove this line? Since it's the same image as on the first line

Reply:
https://github.com/0xSpaceShard/cairo-cli-docker/pull/19/commits/495d1553932745eef246d6013af98c9ca55441af